loadbalanced-fargate-svc/service/pipeline_infrastructure/cloudformation.yaml

```diff
@@ -594,7 +594,8 @@ Resources:
                   - PipelineBuildCodePipelineActionRole
                   - Arn
               RunOrder: 1
-          Name: Build {%- for service_instance in service_instances %}
+          Name: Build 
+{%- for service_instance in service_instances %}
```

Comment:
will this result in an empty line in the rendered template?

Reply:
Sorry I missed this review/discussion back then. Mh... I am using it [here](https://github.com/mreferre/aws-proton-tutorial/blob/main/vpc-ecscluster-lb-fargate-tutorial/lb-fargate-svc/v1/pipeline_infrastructure/cloudformation.yaml#L597-L598) and it seems to be working fine. It was just a readability thing in that it was hard to visually parse where the for loop began and so I created a new dedicated line for the `for` loop. The behavior did not change based on my tests.